reference/5.1/Microsoft.PowerShell.Core/Enter-PSSession.md

```diff
@@ -89,6 +89,10 @@ interactive session. However, you cannot use the `Disconnect-PSSession`, `Connec
 To end the interactive session and disconnect from the remote computer, use the `Exit-PSSession`
 cmdlet, or type `exit`.
 
+> [!IMPORTANT]
+> `Enter-PSSession` is designed to substitute a new remote session for the current interactive
+> session. You shouldn't call it from within script context.
```

Comment:
Thanks for the quick turnaround.

I assume that "a" is missing from "from within script context",
though I suggest mentioning the CLI also, along the lines of:

"You shouldn't call it from a script file or via the PowerShell CLI."